LambdaEngine/Source/Networking/API/PacketTranscoder.cpp

```diff
@@ -3,6 +3,8 @@
 
 #include "Log/Log.h"
 
+#include "Engine/EngineLoop.h"
```

Comment:
Not needed. (Noticed it was for debugging, can stay if that debugging is needed in the future)